test/io/vass/time-write.skipif

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+import os
+
```

Comment:
It may be worth documenting why this skipif exists as a doc-string at the top of this file.